bundle/bundle.go

```diff
@@ -287,12 +291,22 @@ func ValidBundle(bundle Bundle) error {
 	var totalSum int64
 
 	sigs := make(map[Hash][]Trytes)
+	changes := map[trinary.Trytes]int64{}
 	k := kerl.NewKerl()
 
 	lastIndex := uint64(len(bundle) - 1)
 	for i := range bundle {
 		tx := &bundle[i]
+
 		totalSum += tx.Value
```

Comment:
Theoretically, this could also overflow, but for this `tx.Value` would need to be greater than the supply.
I am not 100% how the official validation rules are for transactions with value larger than the supply. So, either check tx.Value against the supply or check for overflows in the addition (maybe just using `big.Int` would be easiest in this case)

Reply:
Value is already checked during the parsing of the transaction. IMHO there is no need to check again?

Reply:
But maybe the the transaction was modified after, created via API/pure client code, ...
It is not checked in this function and if the validation rules forbid to have values > supply that is such an easy check to add making it much more robust without any drawback whatsoever

Reply:
done!